src/dds/rs-dds-sensor-proxy.cpp

```diff
@@ -302,34 +299,39 @@ void dds_sensor_proxy::handle_video_data( realdds::topics::image_msg && dds_fram
 }
 
 
-void dds_sensor_proxy::handle_motion_data( realdds::topics::image_msg && dds_frame,
+void dds_sensor_proxy::handle_motion_data( realdds::topics::imu_msg && imu,
                                            const std::shared_ptr< stream_profile_interface > & profile,
                                            streaming_impl & streaming )
 {
     frame_additional_data data;  // with NO metadata by default!
-    data.timestamp
-        = static_cast< rs2_time_t >( realdds::time_to_double( dds_frame.timestamp ) * 1e-3 );  // milliseconds
-    data.timestamp_domain;  // from metadata, or leave default (hardware domain)
-    data.depth_units;       // from metadata
-    data.frame_number;      // filled in only once metadata is known
+    data.timestamp               // in ms
+        = static_cast< rs2_time_t >( realdds::time_to_double( imu.timestamp() ) * 1e3 );
+    data.timestamp_domain = RS2_TIMESTAMP_DOMAIN_GLOBAL_TIME;  // leave default (hardware domain)
```

Comment:
Comment about HW domain does not match actual "global time" value

Reply:
I'll take out the "global time" setting and leave at default. I changed it when trying stuff.

Bottom line:
Camera-generated domains will always be HW times and, when global time is enable, converted.
Adapter-sent domains may already be global (from the adapter's side) but will be treated as HW meaning may get converted to the client's global time.